templates/README.md

```diff
@@ -0,0 +1,13 @@
+# Templates
+
+This folder contains default templates notably used for "book matters" (cover pages, title and half-title pages, endpapers, etc.).
```

Comment:
On the absence of "License grants for submitted templates" in CONTRIBUTING.
= Indeed, should be added there too.

Reply:
Updated